### DISCUSSION TITLE
New user, pi4 losing 2/3 wans after reboot

### DISCUSSION THREAD

**Discussion by serlowenbrau** [Ysurac/openmptcprouter#2603] [General]
Hello, I am experimenting with replacing my pi/speedify setup to see if openmptcp works better.  The speedify performance wasn't quite what I wanted on PI, so I moved it to an odroid.  That works better (and doesnt' lock up every 2-3 days), but the speedify service still seems to bog down in peak hours, even when the individual connections are not.  I decided to repurpose the pi4 as an omr and see if this works better.  I tried to use the recommended setup using a switch, but I do not have a way to turn off dhcp on my provided lte gateways.  I got nowhere fast trying that way.  Instead, I have 3 usb/ethernet adapters going through a usb hub into the pi4.  This method works for speedify, so I know the adapters are good and I don't have a power issue.  I must have something set up incorrectly.

I have ATT fixed wireless set up as the master connection (gw 176.16.0.1).  I have a new tmobile 5g router set up for connection 2 (192.168.12.1).  I have a tmobile LTE mvno for connection 3 (192.168.10.1).  I left the omr as 198.168.100.1.  I set up an address on each network for each connection (176.16.1.200, 192.168.12.200, 192.168.10.200).  I had the eth0 running directly into my laptop, everything was working great.  I switched the eth0 connection over to my aplifi wifi router, everything still great.  I set up the omr-bypass and got netflix working.  I decided to try turning on sqm on my connections, and then did a reboot.  Nope.  ATT master connection comes up fine, but the other 2 connections will not come back up for anything.  I undid the sqm changes, multiple reboots, still broken.  I have been business programming for 20+ years, but networking has never been my gig.  I have probably done a rookie networking mistake somewhere, so I am hoping someone here can point it out.  

Thanks,
James

**Comment by Ysurac** [comment]
Check System->OpenMPTCProuter "status" page and Status->System log.

**Comment by serlowenbrau** [comment]
Thank you for your quick response yesterday.  My time window to play with the internet had closed... my wife was awake.   lol

Two of the adapters had randomly flipped wan numbers after the reboot.  I restarted from scratch again today, and the 2 wans would still not come up.  Change ethernet connections between the two adapters, everything is suddenly working again.  What is the easiest way to see the mac address of each wan?  I will mark the last segment of the mac on the usb/ethernet adapters with a sharpie, so I can fix this faster if they scramble again.

**Reply by Ysurac** [reply]
You can check if "Disable interfaces auto rename" is unchecked in System->OpenMPTCProuter, "Advanced settings" tab. This should rename device if they are not in same position after a reboot.
To check MAC you can use Network->Interfaces, "Devices" tab.

**Comment by serlowenbrau** [comment]
Thank you very much, I will try unchecking the box.

This project is great.  I very much appreciate your work.
